src/ceph-disk/ceph_disk/main.py

```
@@ -4524,9 +4527,35 @@ def list_devices():
     return devices
 
 
+def list_zfs():
+    try:
+        out, err, ret = command(
+            [
+                'zfs',
+                'list',
+                '-o', 'name,mountpoint'
+            ]
+        )
+    except subprocess.CalledProcessError as e:
+        LOG.info('zfs list -o name,mountpoint '
+                 'fails.\n (Error: %s)' % e)
```

need to re-raise this exception, you can just add
```py
raise
```
in the `except` block.